logrotation/README.md

```diff
@@ -0,0 +1,35 @@
+# Enable Log Rotation for Components
+
+Log rotation options for components
```

Comment:
Why not decide on one approach & propose sample solution for decided one. Any one doesnt suffice all, so two solutions? Can we list down such limitations here, like which component needs what ?

Reply:
Components like motr, s3 use shell scripts for log rotation and hare, csm use logrotate config that's why enabling both ways.